_posts/2019-12-20-notes-from-community-call-19-december-2019.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Notes from community call - 19 December 2019"
+date: 2019-12-20
+author: Jan Ainali
+type: blogpost
+excerpt: We went meta and discussed the community calls themselves.
+categories:
+  - Community call
+---
+
+# Notes from the 19 December 2019 Community Call
+
+## Attendees
+
+* [Boris](https://publiccode.net/team/boris-van-hoytema.html)
+* [Jan](https://publiccode.net/team/jan-ainali.html)
+* [Elena](https://publiccode.net/team/elena-findley-de-regt.html)
+* [Claus](https://publiccode.net/team/claus-mullie.html)
+
+## Update from the Foundation
+
+* Our [blog](https://blog.publiccode.net/) is up and running and news will be posted there more regularly from now on.
+* We got a prototype of the [Governance game](https://github.com/publiccodenet/governance-game).
+* We are going to Connected Smart Cities Conference 23 January and [have a session on governance](https://oasc.events.idloom.com/connected-smart-cities-conference-2020/pages/programme).
+* The office will be closed from 21 December to 5 January.
+
+## Call notes
+
+### Community calls - Background
+
+The community calls have been run biweekly, with topics from us or generated by the community. Our original plan was to have one call per month focus on the Standard for Public Code and one on the Foundation, with different audiences and topics. That distinction has blurred over time, so we've just had biweekly calls.
+
+### Where do we go now?
+
+During the call different options were discussed and weighed against each other. In the end, we decided to stick with the original plan. With our new management of the invites through Odoo, we decided to add an extra tag and assign it to all that are signed up now. That way it is easy to unsubscribe people from one while no one is missing out because of our more explicit division of topics. Jan will organize the Standard call, Elena organizes the Foundation call. The calls will be on the 1st and 3rd Thurs of the month respectively.
```

Comment:
```suggestion
During the call different options were discussed and weighed against each other. In the end, we decided to stick with the original plan. With our new management of the invites through Odoo, we decided to add an extra tag and assign it to all that are signed up now. That way it is easy to unsubscribe people from one while no one is missing out because of our more explicit division of topics. Jan will organize the Standard call, Elena organizes the Foundation call. The calls will be on the 1st and 3rd Thursdays of the month respectively.
```